numpy/random/_generator.pyx

```diff
@@ -4398,6 +4398,9 @@ cdef class Generator:
             char* x_ptr
             char* buf_ptr
 
+        if isinstance(x, memoryview):
```

Comment:
I'm guessing this conditional could just be removed? --was being conservative I suppose

Reply:
Since `x` can be a `list` that we definitely do not want to view as an `ndarray`, this conditional is not optional.